### DISCUSSION TITLE
$app.cache

### DISCUSSION THREAD

**Discussion by ilteoood** [pocketbase/pocketbase#3256] [Q&A] [answered]
Looking at the [intefaces](https://pocketbase.io/jsvm/interfaces/store.Store.html) I see that there's a `cache` function available in $app.
The documentation seems to not mention it. Is there a way to make the cache items expire after a minute, or something like this?

**Comment by ganigeorgiev** [comment, marked as answer]
No, there are no auto eviction policies. Store items can be added or removed only manually at the moment.

**Reply by ilteoood** [reply]
@ganigeorgiev sorry to resurrect this thread, but why is store not in the documentation?
Are you maybe planning to drop support for it?

**Reply by ganigeorgiev** [reply]
No, it is just missing. The documentation will be updated after the [ongoing refactoring](https://github.com/pocketbase/pocketbase/discussions/4355).
